libs/sdk-ui-kit/styles/scss/chartSorting.scss

```diff
@@ -0,0 +1,149 @@
+// (C) 2022 GoodData Corporation
+@import "./Dialog/_mixins";
+@import "./variables";
+@import "./mixins.scss";
+@import "./Button/_variables";
+
+$measure_sequence_number_font: menlo, monaco, consolas, "Courier New", monospace;
```

Comment:
can we keep only this definition and use it in rakingFilter as imported? To keep single place of definition